src/reducers/index.js

```diff
@@ -22,6 +22,16 @@ function saveData(state, value) {
 }
 
 function clearSearch(state) {
+    fetch("https://api.openbrewerydb.org/breweries/")

```

Comment:
`clearSearch` and `loadData` contains almost the same logic(code), please refactor them to fix this DRY issue